src/commands/workflow/run.ts

```diff
@@ -21,13 +22,18 @@ export default class WorkflowRun extends BaseCommand {
     actor: Flags.string(),
     tenant: Flags.string(),
     data: jsonStr(),
+    "run-local": Flags.boolean({ default: false }),
   };
 
   static args = [{ name: "workflowKey", required: true }];
 
   async run(): Promise<void> {
     const { args, flags } = this.props;
 
+    if (flags["run-local"]) {
```

Comment:
Super nit, feel free to ignore — it seems like the only part that's different is the `workflow` param, whether we need to read from the workflow dir or not. 

In that case, I may re-organize the code so that becomes clearer. Something like:

```
const workflow = flags["run-local"]
  ? await this.getWorkflowInput() 
  : undefined

// .. the rest is the same for making a request to run a remote workflow vs a local workflow.
```